keyboards/clawsome/luggage_rack/info.txt

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Wrong filetype, should be `info.json`.

Reply:
Still outstanding.

Reply:
tried to adjust from .txt to .json, ended up pulling a bunch of random files during a rebase.

what should I do?

Reply:
Just rename the file? You don't need to do any rebasing.

Reply:
It did that thing where my local GitHub Desktop decided to "adjust" some 1900 files and count that towards my file count. I made it undo that today.

Apparently I also had an 'info.txt' and an 'info.json' in there, so I just deleted the former.